### PR TITLE
Add constants for tool names

### PR DESCRIPTION
**Pull request by mnonnenmacher**
This ensures consistent use of the same name for the tools within code.
Also introduce a constant where the string is only used once for
consistency.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/168)

### CODE REVIEW

File: analyzer/src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -45,6 +45,8 @@ import kotlin.system.exitProcess
  * The main entry point of the application.
  */
 object Main {
+    val TOOL_NAME = "analyzer"
```
Comment (sschuberth) [inline analyzer/src/main/kotlin/Main.kt]: All these can be `const val`.

Comment (mnonnenmacher) [reply analyzer/src/main/kotlin/Main.kt]: Done.

**Comment by sschuberth** [comment]
@mnonnenmacher, you may also want to use the constant here:

    analyzer/src/main/kotlin/PackageManager.kt:        val HTTP_CACHE_PATH = "analyzer/cache/http"

**Comment by sschuberth** [comment]
Ping @mnonnenmacher, you did see my comment above, right? 😉

**Comment by mnonnenmacher** [comment]
Yes, just opened a PR for it.

**Pull request merged**